elements/castbar.lua

```diff
@@ -142,13 +167,9 @@ local function UNIT_SPELLCAST_START(self, event, unit)
 		shield:Hide()
 	end
 
-	local sf = element.SafeZone
-	if(sf) then
-		sf:ClearAllPoints()
-		sf:SetPoint(element:GetReverseFill() and 'LEFT' or 'RIGHT')
-		sf:SetPoint('TOP')
-		sf:SetPoint('BOTTOM')
-		updateSafeZone(element)
+	if (element.SafeZone) then
+		positionSafeZone(element);
+		updateSafeZone(element);
```

Comment:
Too much PHP/JS lately? :P

Reply:
Heh? :)

Reply:
Hahahah you got me with those semi-colons